OptimizelySDK.Tests/DecisionServiceTest.cs

```diff
@@ -27,30 +27,41 @@ namespace OptimizelySDK.Tests
 {
     public class DecisionServiceTest
     {
-
         private string GenericUserId = "genericUserId";
         private string WhitelistedUserId = "testUser1";
         private string UserProfileId = "userProfileId";
         private Mock<ILogger> LoggerMock;
         private Mock<IErrorHandler> ErrorHandlerMock;
         private Mock<UserProfileService> UserProfileServiceMock;
+        private Mock<Bucketer> BucketerMock;
+        private Mock<FeatureFlag> FeatureFlagMock;
+        private Mock<DecisionService> DecisionServiceMock;
 
         private ProjectConfig NoAudienceProjectConfig;
         private ProjectConfig ValidProjectConfig;
+        private ProjectConfig ProjectConfig;
         private Experiment WhitelistedExperiment;
         private Variation WhitelistedVariation;
+        private DecisionService DecisionService;
 
         [SetUp]
         public void SetUp()
         {
-            LoggerMock              = new Mock<ILogger>();
-            ErrorHandlerMock        = new Mock<IErrorHandler>();
-            UserProfileServiceMock  = new Mock<UserProfileService>();
-
-            ValidProjectConfig          = ProjectConfig.Create(TestData.ValidDataFileV3, LoggerMock.Object, ErrorHandlerMock.Object);
-            NoAudienceProjectConfig     = ProjectConfig.Create(TestData.NoAudienceProjectConfigV3, LoggerMock.Object, ErrorHandlerMock.Object);
-            WhitelistedExperiment       = ValidProjectConfig.ExperimentIdMap["223"];
-            WhitelistedVariation        = WhitelistedExperiment.VariationKeyToVariationMap["vtag1"];
+            LoggerMock = new Mock<ILogger>();
```

Comment:
Please place your lines, before or after the already checked-in lines. It's difficult to know the difference what you are adding in this method.

Reply:
Done